lib/ripper-tags/data_reader.rb

```diff
@@ -75,6 +71,21 @@ def resolve_file(file, depth = 0, &block)
       end
     end
 
+    def resolve_files_in_directory(directory, depth, &block)
+      unless File.readable?(directory)
```

Comment:
Instead of checking `File.readable?` could we handle an exception from `Dir.entries`?

Reply:
Thanks for the review - sure, I've pushed a fixup to do that